docs/deployment/getting-started.md

```diff
@@ -0,0 +1,180 @@
+# Deployment
+
+Server deployment is a complex area, that will depend on what kind of service you're deploying Uvicorn onto.
+
+### TL;DR
+As a general rule, you probably want to:
+
+* Run `uvicorn --reload` from the command line for local development.
+* Run `gunicorn -k uvicorn.workers.UvicornWorker` for production (non containerized).
+* Run behind Nginx for self-hosted deployments.
+* Finally, run everything behind a CDN for caching support, and serious DDOS protection.
+
+## Quick Start
+
+In the Quick Start section, we will cover how to start serving your application. The application mentioned here is any ASGI compatible application. For instance if an app that was built using a framework like `Starlette` or `FastAPI`.
+
+The following are ideal for running your application in a development environment.
+
+### Running from the command line
+
+The simplest way to run `uvicorn` is from the command line.
+
+```bash
+$ uvicorn example:app --reload --port 5000
+```
+
+The ASGI application should be specified in the form `path.to.module:instance.path`
```

Comment:
What does `instance.path` mean here? As far as I know it's just `path.to.module:asgi_app_global`, right?

Reply:
I got this referring from either existing documentation or from FastAPI. It was supposed to mean `main:app` or `app.main:app`

Reply:
Yeah those sound right but `module:instance.path` does not

Reply:
Agreed. I'll change that!